doc/src/build/wallet.md

```diff
@@ -9,27 +9,28 @@ Sui wallet and execute wallet commands through its command line
 interface, *Wallet CLI*.
 
 
-## Setup
+## Set up
```

Comment:
is this change intended? in `tutorials.md` we use Setup, we need to keep it consistent?

Reply:
Yes and good catch. I've now fixed this in PR #1038 1038. We are using "setup" like a verb, but that's a noun. "Set up" is a verb.